### PR TITLE
feat: add Transcoder connector sample

### PR DESCRIPTION
**Pull request by j9che**
Docs: add a sample workflow for Transcoder connector

This is used for the Transcoder connector documentation.

CloudWorkflows team will be responsible for updating/maintaining this sample.

### CODE REVIEW

File: src/connectors/connector_transcoder.workflows.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+      },
+      {
+        "the_end": {
+          "return": "${response}"
+        }
+      }
+    ]
+  }
+}
```
Comment (rogerthatdev) [inline src/connectors/connector_transcoder.workflows.json:32]: just add a new line at the end of this yaml

**Review by rogerthatdev — APPROVED** [review]
Verified that the workflow deploys successfully

File: src/connectors/connector_transcoder.workflows.json (hunk 1 above)
Comment (j9che) [reply src/connectors/connector_transcoder.workflows.json:32]: This Json is auto-generated by the tojson.sh script. I think we can leave it unchanged to match existing patterns. Thanks.

**Comment by j9che** [comment]
@rogerthatdev Can you merge this PR? I don't have write access to the repo. Thanks!

**Pull request merged**